src/test/data/JsonSerializableTaAssistTest/duplicateStudentTaAssist.json

```diff
@@ -9,7 +9,8 @@
     "name": "Alice Pauline",
     "phone": "94351253",
     "email": "pauline@example.com",
-    "address": "4th street"
+    "address": "4th street",
+    "classes": [ "family" ]
   } ],
-  "moduleClasses" : []
+  "moduleClasses" : [ "friends", "family" ]
```

Comment:
We might want to change the "classes" attribute here to actual classes instead of reusing the previous tags examples. Not a big issue and will get it fixed in future issues :)

Reply:
Can be updated closer to finalizing the product.